x-pack/test/apm_api_integration/tests/diagnostics/apm_events.spec.ts

```diff
@@ -0,0 +1,131 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+import { apm, timerange } from '@kbn/apm-synthtrace-client';
+import { FtrProviderContext } from '../../common/ftr_provider_context';
+
+export default function ApiTest({ getService }: FtrProviderContext) {
+  const registry = getService('registry');
+  const apmApiClient = getService('apmApiClient');
+  const es = getService('es');
+  const synthtraceEsClient = getService('synthtraceEsClient');
+
+  const start = new Date('2021-01-01T00:00:00.000Z').getTime();
+  const end = new Date('2021-01-01T00:15:00.000Z').getTime() - 1;
+
+  registry.when('Diagnostics: APM Events', { config: 'basic', archives: [] }, () => {
+    describe('When there is no data', () => {
+      before(async () => {
+        // delete APM data streams
+        await es.indices.deleteDataStream({ name: '*apm*' });
+      });
+
+      it('returns zero data streams`', async () => {
+        const { status, body } = await apmApiClient.adminUser({
+          endpoint: 'GET /internal/apm/diagnostics',
+        });
+        expect(status).to.be(200);
+        expect(body.apmEvents.every(({ docCount }) => docCount === 0)).to.be(true);
+      });
+    });
+
+    describe('When data is ingested', () => {
+      before(async () => {
+        const instance = apm
+          .service({ name: 'synth-go', environment: 'production', agentName: 'go' })
+          .instance('instance-a');
+
+        await synthtraceEsClient.index(
+          timerange(start, end)
+            .interval('1m')
+            .rate(30)
+            .generator((timestamp) =>
+              instance
+                .transaction({ transactionName: 'GET /users' })
+                .timestamp(timestamp)
+                .duration(100)
+                .success()
+            )
+        );
+      });
+
+      after(() => synthtraceEsClient.clean());
+
+      it('returns zero doc_counts when no time range is specified', async () => {
```

Comment:
Couldn't we just set a default time range here? and override if a specific one is set?

Reply:
If not time range is given, we query the past 5 minutes. In this case data is ingested in 2021.